controller/persistence/asynchronous_event_store.go

```diff
@@ -0,0 +1,99 @@
+/*
+	Copyright NetFoundry, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package persistence
+
+import (
+	"github.com/openziti/foundation/storage/ast"
+	"github.com/openziti/foundation/storage/boltz"
+	"go.etcd.io/bbolt"
+)
+
+const (
+	FieldAsynchronousEventType = "type"
+	FieldAsynchronousEventData = "data"
+)
+
```

Comment:
Wouldn't hate it if we used AsyncEvent instead :) I don't think we lose any clarity with the shorter name

Reply:
Was going to change it to "EventualEventer" because it isn't 100% async. But year the name is long.